### PR TITLE
AC Automated Test - All tests from the LRAC-10263 group | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10265
https://issues.liferay.com/browse/LRAC-10266
https://issues.liferay.com/browse/LRAC-10269
https://issues.liferay.com/browse/LRAC-10270
https://issues.liferay.com/browse/LRAC-10271
https://issues.liferay.com/browse/LRAC-10273
https://issues.liferay.com/browse/LRAC-10274
https://issues.liferay.com/browse/LRAC-10275
https://issues.liferay.com/browse/LRAC-10276
https://issues.liferay.com/browse/LRAC-10277
https://issues.liferay.com/browse/LRAC-10278
https://issues.liferay.com/browse/LRAC-10279
https://issues.liferay.com/browse/LRAC-10280
https://issues.liferay.com/browse/LRAC-10281
https://issues.liferay.com/browse/LRAC-10282
https://issues.liferay.com/browse/LRAC-10283
https://issues.liferay.com/browse/LRAC-10286
https://issues.liferay.com/browse/LRAC-10287
https://issues.liferay.com/browse/LRAC-10267
https://issues.liferay.com/browse/LRAC-10268
https://issues.liferay.com/browse/LRAC-10272
https://issues.liferay.com/browse/LRAC-10316

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsFilterBreakdown.testcase [hunk 1]
```diff
@@ -123,7 +124,957 @@ definition {
...
+				attributeName = "${attributeName}",
+				eventName = "${customEventName}",
+				filterName = "${filterName}");
+		}
+
+		task ("Remove the attribute from the analysis") {
+			ACEventAnalysis.removeFromAnalysis(removeValue = "${customEventName}");
+		}
+
+		task ("Check that the analysis dashboard contains the event,attribute and filter") {
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsFilterBreakdown.testcase:428]: Hi @ViniLopes20, Change the task to:

`task ("Check that the analysis dashboard does not contain the event, attribute and filter")`

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsFilterBreakdown.testcase [hunk 2]
```diff
@@ -123,7 +124,1024 @@ definition {
...
+				commonInformation = "site name",
+				index = "3",
+				informationValue = "custom 1 - site name - ${dataSource}");
+		}
+	}
+
+	@description = "Feature ID: LRAC-7868 | Automation ID: LRAC-10267 | Test Summary: Rearrange breakdown and assert data change and after rearrange to return to original state"
+	@priority = "4"
+	test AssertDataChangeWhenRearrangeBreakdownAndReturnOriginalState {
+		var attributeNameList = "category,pageTitle,url";
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsFilterBreakdown.testcase:261]: Hi @ViniLopes20 , in this test case you also need to validate the data changes after changing the order of the attribute breakdown, not only the position of the attributes.

Maybe the macro `viewAnalysisInformationInOrder` could be used in addition to the position validations you are already doing here.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsFilterBreakdown.testcase [hunk 3]
```diff
@@ -123,7 +124,1024 @@ definition {
...
+				index = "3");
+		}
+
+		task ("Change the breakdown position order") {
+			ACEventAnalysis.changeBreakdownPosition(
+				indexEnd = "1",
+				indexStart = "3");
+		}
+
+		task ("Check the order of the columns of the analysis has changed") {
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsFilterBreakdown.testcase:337]: Hi @ViniLopes20 , in this test case you also need to validate the data changes after changing the order of the attribute breakdown, not only the position of the attributes.

Maybe the macro viewAnalysisInformationInOrder could be used in addition to the position validations you are already doing here.

**Comment by alexcandidos** [comment]
Good job, @ViniLopes20 !!

Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Pull request closed without merging**